config/samples/mellanox.com_v1alpha1_ipoibnetwork.yaml

```diff
@@ -20,10 +20,6 @@ spec:
   master: "ibs3f1"
   ipam: |
     {
-      "type": "whereabouts",
-      "range": "192.168.6.225/28",
-      "exclude": [
-       "192.168.6.229/30",
-       "192.168.6.236/32"
-      ]
+      "type": "nv-ipam",
+      "poolName": "my-pool"
```

Comment:
We also need to add a sample of the nv-ipam pool here and in other examples where it's missing

Reply:
We've got it here: https://github.com/Mellanox/network-operator/blob/master/example/crs/nvidia.com_v1alpha1_nv_ipam_ippool_cr.yaml. I don't want to copy-paste it